src/Form/ConfigForm.php

```diff
@@ -84,12 +84,23 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
 
     $form['access_token_ttl'] = [
       '#type' => 'number',
-      '#title' => $this->t('Access Token Expiration (in seconds)'),
+      '#title' => $this->t('Access Token Expiration'),
       '#default_value' => $cool_settings['access_token_ttl'] ?? 0,
+      '#field_suffix' => $this->t('seconds'),
       '#min' => 0,
       '#required' => TRUE,
     ];
 
+    $form['new_file_interval'] = [
+      '#type' => 'number',
+      '#title' => $this->t('New File interval'),
```

Comment:
For the form element label I don't like it. What about
```suggestion
      '#title' => $this->t('Create new file on save after…'),
```